.travis.yml

```diff
@@ -74,6 +78,39 @@ jobs:
         - export NUMBA_DISABLE_JIT=1; doit test_unit_nojit
 
 
+    ########## DOCS ##########
+
+    - &doc_build
+      <<: *conda_default
+      stage: docs
+      env: HV_DOC_HTML="true" DESC="docs" CHANS_DEV="-c pyviz/label/dev"
+      script:
+        - doit develop_install $CHANS_DEV -o doc -o examples_extra
+        - datashader fetch-data --path=examples
+        - nbsite generate-rst --org pyviz --project-name datashader --skip '.*tiling.*,.*streaming-aggregation.*'
+        - nbsite build --what=html --output=builtdocs
+      deploy:
+        provider: pages
+        skip_cleanup: true
+        github_token: $GITHUB_TOKEN
+        local_dir: ./builtdocs
+        fqdn: datashader.org
+        on:
+          tags: true
+          all_branches: true
+
+    - <<: *doc_build
+      stage: docs_dev
+      deploy:
+        provider: pages
+        skip_cleanup: true
+        github_token: $GITHUB_TOKEN
+        local_dir: ./builtdocs
+        repo: pyviz-dev/datashader
+        on:
+          tags: true
+          all_branches: true
+
```

Comment:
It looks like there is no longer an obvious way to build the docs locally?  I do want them to be built by CI in the typical case, but the point of using dodo.py was to make it simple to run the same command that CI runs, so that both local and CI builds are based on the same spec.

Reply:
yes the problem is that with dodo the output is all dumped out at the end, so travis can time out and it is really hard to tell what is going on.

Reply:
Ugh.